pkg/executables/clusterctl.go

```diff
@@ -148,12 +155,43 @@ func writeInfrastructureBundle(clusterSpec *cluster.Spec, rootFolder string, bun
 	return nil
 }
 
+func ClusterctlMoveRetryPolicy(totalRetries int, err error) (retry bool, wait time.Duration) {
```

Comment:
Is there a need to export this?

Reply:
I'm also wondering if this could be a generalized exponential backoff and live in the `retrier` package?

Reply:
Yea this looks generic enough to maybe warrant a generic retry policy, and we can pass in the expected error too?

Reply:
Agreed that this is possible, and would be DRYer.  This policy, and the one I created for kubectl wait, are near identical.  

The technique they use, of regex matching the the error.Error() string is distinctive to use cases where there's no more reliable way to identify the kind of error, and TBH kind of icky, so I'm hesitant to factor them out, as it implies that this is the a recommended technique for how retry policies should work.  I'd want the recommended technique uses distinctly enumerated error codes.  Keeping these "aberrations" sequestered in the modules that use them hints that they're very module-specific.

I revisited kubectl wait, and am reminded of the technique I used for aliasing an unexported method for testing (export_test.go).  So I can make this more private if still desired.

Reply:
I empathize with the error handling issue, we've not been great at managing errors. If we could unexport it we can refactor when we have better error handling. I'm not sure what technique you're referencing, but if you need any input let me know - I have several tricks.

Reply:
I would probably just move this method to `pkg/executables/internal` so we can keep the unit test as is

Reply:
Thanks @chrisdoherty4 and @g-gaston.  I chose to follow the same approach I'd used recently kubectl wait (but apperently forgot about!): export clusterctlMoveRetryPolicy as ClusterctlMoveRetryPolicy in export_test.go.  Since this is a *test.go file, it's not included in non-test builds, so the name remains unexported there.  (Following from https://medium.com/@robiplus/golang-trick-export-for-test-aa16cbd7b8cd)